src/plugins/data_views/public/data_views/data_views_api_client.ts

```diff
@@ -16,11 +16,16 @@ const API_BASE_URL: string = `/api/index_patterns/`;
 const version = '1';
 
 async function sha1(str: string) {
-  const enc = new TextEncoder();
-  const hash = await crypto.subtle.digest('SHA-1', enc.encode(str));
-  return Array.from(new Uint8Array(hash))
-    .map((v) => v.toString(16).padStart(2, '0'))
-    .join('');
+  try {
+    const enc = new TextEncoder();
+    const hash = await crypto.subtle.digest('SHA-256', enc.encode(str));
+    return Array.from(new Uint8Array(hash))
+      .map((v) => v.toString(16).padStart(2, '0'))
+      .join('');
+  } catch (error) {
+    const { sha256 } = await import('./sha256');
+    return sha256(str);
+  }
```

Comment:
Nit: I wonder if there's a clearer way of testing for availability of the `crypto` API rather than try/catch, like testing if `crypto.subtle.digest` is defined or not? Not something we need to address in this PR, just thinking out loud.

Reply:
Good call! I wasn't happy about the try/catch but was overlooking the obvious